### DISCUSSION TITLE
Port to Linux

### DISCUSSION THREAD

**Discussion by mardzie** [nameouschangey/Stormworks_VSCodeExtension#136] [Ideas]
Hi,
could you please port this awesome extension to Linux? I love Stormworks and I love Linux and this extension is so helpful when creating systems in Lua.

Thank you

**Comment by Chromatischer** [comment]
Hey, I am actively working on a port for neovim for this plugin, I will get back here once I am finished. Do you have any idea on how I could get into contact with @nameouschangey to ask for his permission on publishing?

**Reply by mardzie** [reply]
That would be awesome. I sadly don't have any idea how to contact @nameouschangey. He seems pretty inactive so he probably won't see this discussion either.

**Reply by Chromatischer** [reply]
Yea, I have not tested anything for SW-Addons and still have some issues with compiling but I wish to be done later this week. Furthermore there currently is no support for the Emulator
